flocker_bb/builders/maint.py

```diff
@@ -232,7 +232,10 @@ def _destroy_cloud_volumes(self, volumes):
         Unconditionally and irrevocably destroy all of the given cloud volumes.
         """
         for volume in volumes:
-            volume.destroy()
+            try:
+                volume.destroy()
+            except:
+                pass
```

Comment:
As you mentioned, it'd be nice to at least log the exception so we at least know it happened, There's probably a quick way to at least do that?